_posts/2020-10-21-explicit-term-inference-in-scala-3.md

```diff
@@ -0,0 +1,433 @@
+---
+layout: blog-detail
+post-type: blog
+by: Meriam Lachkar, Vincenzo Bazzucchi, Scala Center
+title: Explicit term inference with Scala 3
+---
+
+# Explicit term inference with Scala 3
+
+One of the most striking changes for developers adopting Scala 3 is the introduction
+of new syntaxes to replace the implicit mechanism used in previous Scala versions.
+
+The motivation behind the new syntax is that the same `implicit` keyword was
+used for different purposes and patterns and thus it became a way to express *how* to implement
+patterns. This means that when encountering this ambiguous incantation, users need to decipher what
+the intent of the developer was: is this a conversion ? Does this avoid parameter repetition ?
+Is this an extension of a type ? Is this a typeclass ? How do I import this ?
```

Comment:
Conventionally, there would be no space before an `?` in _English_ (unlike French).